interfaces/automated-testing/scd/scd.yaml

```diff
@@ -35,6 +35,402 @@ components:
+            altitude_lower, if specified.
+          anyOf:
+            - $ref: "#/components/schemas/Altitude"
+    Volume4D:
+      description: Contiguous block of geographic spacetime.
+      required:
+        - volume
+      type: object
+      properties:
+        volume:
+          $ref: "#/components/schemas/Volume3D"
+        time_start:
+          description: Beginning time of this volume. Must be before time_end.
+          anyOf:
+            - $ref: "#/components/schemas/Time"
+        time_end:
+          description: End time of this volume. Must be after time_start.
+          anyOf:
+            - $ref: "#/components/schemas/Time"
+
+    UssBaseURL:
+      description: >-
+        The base URL of a USS implementation of part or all of the USS-USS API.
+        Per the USS-USS API, the full URL of a particular resource can be
+        constructed by appending, e.g., `/uss/v1/{resource}/{id}` to this base
+        URL. Accordingly, this URL may not have a trailing '/' character.
+      type: string
+      example: https://uss.example.com/utm
+
+    UssAvailabilityState:
+      type: string
+      enum:
+        - Unknown
+        - Normal
+        - Down
+      description: >-
+        A USS is presumed to be in the Unknown state absent indication otherwise
+        by a USS with availability arbitration scope.  Upon determination via
+        availability arbitration, a USS is Down when it does not respond
+        appropriately, and a Down USS may not perform the following operations
+        in the DSS:
+        * Create an operational intent in the Accepted or Activated states
+        * Modify an operational intent whose new or unchanged state is Accepted
+        or Activated
+        * Delete an operational intent
+        A USS in the Unknown state possesses all the capabilities, within the
+        DSS, of a USS in the Normal state.
+    EntityOVN:
+      title: EntityOVN
+      description: >-
+        A token associated with a particular UTM Entity+version created by the
+        DSS upon creation or modification of an Entity reference and provided to
+        the client creating or modifying the Entity reference.  The EntityOVN is
+        stored privately by the DSS and then compared against entries in a Key
+        provided to mutate the airspace.  The EntityOVN is also provided by the
+        client whenever that client transmits the full information of the Entity
+        (either via GET, or via a subscription notification).
+      type: string
+      minLength: 16
+      maxLength: 128
+      example: 9d158f59-80b7-4c11-9c0c-8a2b4d936b2d
+    SubscriptionID:
+      description: >-
+        Identifier for a subscription communicated through the DSS.  Formatted
+        as a UUIDv4.
+      anyOf:
+        - $ref: "#/components/schemas/UUIDv4Format"
+      example: 78ea3fe8-71c2-4f5c-9b44-9c02f5563c6f
+
+
+    OperationalIntentUssBaseURL:
```

Comment:
This schema is not used in this interface. It can be removed.

Reply:
Thanks, removed now.